thavalon-webapp/src/components/Register.tsx

```diff
@@ -1,32 +1,88 @@
-import React, {useEffect, useState} from 'react';
+import React, {useState} from 'react';
 import ReactModal from 'react-modal';
 import { useForm } from 'react-hook-form';
 import "./modal.scss";
-import { register_user } from '../utils/account_utils';
+import AccountManager, { HttpResponse } from '../utils/accountManager';
+import { Redirect } from 'react-router-dom';
+
+type RegisterProps = {
+    setLoggedIn: any

```

Comment:
Should this setLoggedIn be any type?

Reply:
I think type is dispatch or something? Idk how to type it, will look into it